src/pieces/logs/serverLog.ts

```diff
@@ -283,13 +283,26 @@ async function processInviteDelete(invite: Invite, serverLog: TextChannel): Prom
 
 async function processMessageDelete(msg: Message | PartialMessage, serverLog: TextChannel): Promise<void> {
 	if (!('name' in msg.channel) || msg.guild.id !== GUILDS.MAIN) return;
-	const embed = new MessageEmbed()
-		.setAuthor(msg.author.tag, msg.author.avatarURL({ dynamic: true }))
-		.setTitle(`Message deleted in #${msg.channel.name} | Sent ${msg.createdAt.toLocaleString()} ` +
-			`(${prettyMilliseconds(Date.now() - msg.createdTimestamp, { verbose: true })} ago)`)
-		.setFooter(`Message ID: ${msg.id} | Author ID: ${msg.author.id}`)
-		.setColor('ORANGE')
-		.setTimestamp();
+
+	let embed;
+	if (!msg.author) { // this message is a partial, so author/content information is not available.
```

Comment:
You can check if a message is a partial with `msg.partial` ([see docs](https://discord.js.org/#/docs/discord.js/stable/class/Message?scrollTo=partial))